traffic_portal/app/src/common/modules/table/cacheGroupParameters/table.cacheGroupParameters.tpl.html

```diff
@@ -48,7 +60,6 @@
                 <td data-search="^{{::p.configFile}}$">{{::p.configFile}}</td>
                 <td data-search="^{{::p.value}}$">{{::p.value}}</td>
                 <td data-search="^{{::p.secure}}$">{{::p.secure}}</td>
-                <td><button type="button" class="btn btn-link" title="Unlink Parameter from Cache Group" ng-click="removeParameter(p.id)"><i class="fa fa-chain-broken"></i></button></td>
```

Comment:
why the removal of this? I think it was the easiest way to remove associations without the need to right click or search through the big list

Reply:
this is kind of a "fake" column. it doesn't map to a property of a "parameter" so it makes things messy when defining columns in a dynamic and configurable way. i.e. 


```
			"columns": [
				{ "name": "Name", "visible": true, "searchable": true },
				{ "name": "Config File", "visible": true, "searchable": true },
				{ "name": "Value", "visible": true, "searchable": true },
				{ "name": "Secure", "visible": true, "searchable": true }
			],
```

so rather than deal with it, i just removed it. you can still remove a param from a cg using the "context menu" (right click)

Reply:
would i have liked to leave it? yeah, but wasn't finding a good solution to keep it AND provide configurable show/hide capability of parameter columns....

Reply:
actually, give me a sec. i have an idea.

Reply:
updated